components/admin_console/manage_teams_modal/manage_teams_modal.jsx

```diff
@@ -56,17 +56,11 @@ export default class ManageTeamsModal extends React.Component {
         }
     }
 
-    loadTeamsAndTeamMembers = (user = this.props.user) => {
-        TeamActions.getTeamsForUser(user.id, (teams) => {
-            this.setState({
-                teams: filterAndSortTeamsByDisplayName(teams),
-            });
-        });
-
-        TeamActions.getTeamMembersForUser(user.id, (teamMembers) => {
-            this.setState({
-                teamMembers,
-            });
+    loadTeamsAndTeamMembers = async (user = this.props.user) => {
+        this.getTeamMembers(user.id);
+        const {data} = await this.props.getTeamsForUser(user.id);
+        this.setState({
+            teams: filterAndSortTeamsByDisplayName(data),
```

Comment:
Can these be passed in as props? They should be in the store somewhere

Reply:
Teams are not the same as what we have in state. State will have user teams. Where as the teams here are for other users and used to populate for managing by sys admin.

Reply:
Ah, I see